src/foam/u2/view/ReciprocalSearch.js

```diff
@@ -16,6 +16,18 @@ foam.CLASS({
     'foam.u2.view.SearchViewWrapper'
   ],
 
+  constants: [
+    {
+      name: 'FORMAT_REGEX',
```

Comment:
I don't think you need this. Numbers have a toLocalString method that does this for you.

Reply:
Number(12234234).toLocaleString();   -> "12,234,234"

Reply:
Resolved by 26c02f92336f9547fa1a094d0f70000521674d0c.